src/components/analyses/landing/BatchResults.js

```diff
@@ -0,0 +1,158 @@
+        isFetchingNextPage,
+        fetchNextPage,
+        hasNextPage,
+        error,
+    } = useAnalysesSearchInfinite(
+        [
+            ANALYSES_SEARCH_QUERY_KEY,
+            {
+                rowsPerPage: constants.DEFAULT_PAGE_SIZE,
+                orderBy: analysisRecordFields.START_DATE.key,
+                order: constants.SORT_DESCENDING,
+                filter: JSON.stringify([
+                    { field: "parent_id", value: parentId },
+                ]),
+            },
+        ],
+        enabled,
+        (lastGroup, allGroups) => {
+            const totalPages = Math.ceil(
+                lastGroup?.total / constants.DEFAULT_PAGE_SIZE
+            );
+            if (allGroups.length < totalPages) {
+                return allGroups.length;
+            } else {
+                return false;
+            }
+        }
+    );
+
+    const columns = React.useMemo(
+        () => [
+            {
+                Header: analysisRecordFields.NAME.fieldName,
+                accessor: analysisRecordFields.NAME.key,
+                Cell: ({ row }) => (
+                    <Link
+                        href={`/${NavigationConstants.ANALYSES}/${row?.original.id}?view=${BATCH_DRILL_DOWN}`}
+                    >
+                        <DELink
+                            id={row?.original.id}
+                            text={row?.original.name}
+                            title={row?.original.name}
+                        />
+                    </Link>
+                ),
+            },
+            {
+                Header: analysisRecordFields.STATUS.fieldName,
+                accessor: analysisRecordFields.STATUS.key,
+            },
+        ],
+        [analysisRecordFields]
+    );
+
+    if (error) {
+        return (
+            <ErrorTypographyWithDialog
+                errorMessage={t("errorSearch")}
+                errorObject={error}
+                baseId={baseId}
+            />
+        );
+    }
+    if (
+        status !== constants.LOADING &&
+        (!data ||
+            data.pages.length === 0 ||
+            data.pages[0].analyses.length === 0)
+    ) {
+        return <Typography>{t("noResults")}</Typography>;
+    }
+
+    let flatData = [];
+    if (data && data.pages[0].analyses.length > 0) {
+        data.pages.forEach((page) => {
+            flatData = [...flatData, ...page.analyses];
+        });
+    }
+
+    return (
+        <Accordion
+            expanded={expandListing}
+            onChange={() => setExpandListing(!expandListing)}
+        >
+            <AccordionSummary
+                expandIcon={<ExpandMore />}
+                aria-controls={buildID(baseId, ids.BATCH_PANEL)}
+                id={buildID(baseId, ids.BATCH_PANEL_HEADER)}
+            >
+                <Typography variant="subtitle2" color="primary">
+                    {t("batchDetails")}
+                </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
```

Comment:
Good question but I don't see that in example docs - https://v4.mui.com/components/grid/#grid-with-breakpoints 🤔

Reply:
Must be a typo in their example (I assume you meant at https://v4.mui.com/components/accordion/), because it's setup like this in the examples at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

Reply:
Doh. Wrong link.

Reply:
Oh, I just inspected their example, and it looks like MUI is auto-generating a `div` that wraps the `AccordionDetails` content, and that auto-generated `div` gets the attributes `id="panel1a-content" aria-labelledby="panel1a-header"`.

I guess we only need to manually set the content panel's ID if we're not using an `AccordionDetails` panel.